src/Claims.ts

```diff
@@ -124,7 +132,7 @@ export class Claims {
     const result = await context.queryMiddleware<Ensured<Query, 'didsWithClaims'>>(
       didsWithClaims({
         dids: targets?.map(target => signerToString(target)),
-        scope,
+        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : scope,
```

Comment:
```suggestion
        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : undefined,
```

Just to be more explicit